commitizen/providers.py

```diff
@@ -1,6 +1,7 @@
 from __future__ import annotations
 
 import json
+import jsonpath_ng  # type: ignore
```

Comment:
May I know why do we need `#type: ignore` here

Reply:
Also, may I know why do we need to install this package?

Reply:
I'd also like to know, is it not possible to update it accessing like a json?

Reply:
The problem came from trying to access the correct element in "packages". Here's the sample JSON:

```json
{
  "name": "whatever",
  "version": "0.1.0",
  "lockfileVersion": 2,
  "requires": true,
  "packages": {
    "": {
      "name": "whatever",
      "version": "0.1.0"
    },
    "someotherpackage": {
      "version": "0.1.0"
    }
  }
}
```

The json path expression is `$.packages.''.version`, but I couldn't get that to work via the json object directly. 

And the `# type: ignore` is there b/c the library doesn't appear to have type hints, nor is there anything available for it on typeshed.

Reply:
Okay - so not sure why I was struggling with using the json.loads and updated the dict, but I have that working now. Will remove the dependency on jsonpath_ng.